ethcore/src/state/account.rs

```diff
@@ -479,7 +479,12 @@ impl Account {
 
 impl fmt::Debug for Account {
 	fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-		write!(f, "{:?}", PodAccount::from_account(self))
+		f.debug_struct("Account")
+			.field("balance", &self.balance)
+			.field("nonce", &self.nonce)
+			.field("code", &self.code())
+			.field("storage", &self.storage_changes.iter().collect::<BTreeMap<_, _>>())
+			.finish()
```

Comment:
removes redundant code and storage allocation on debug

Reply:
but the allocation is still present one line above

Reply:
you are right! it's just way smaller, cause the items are not cloned :p updated the description :)